service/src/main/resources/application-user-mappings.yaml

```diff
@@ -0,0 +1,6 @@
+spring:
+  profiles: local,user-mappings
```

Comment:
I see no value in having `user-mappings` profile, a `local` profile is only thing we need.

Reply:
https://stackoverflow.com/questions/35902616/starting-spring-application-by-merging-yml-files i think we must set two different profiles for this as shown here ^ let me know what you think

Reply:
closing as you are happy with approach discussed